index.html

```diff
@@ -81,7 +81,15 @@
 
             <ul id="nav-links" class="nav__links">
                 <li class="nav__item nav__story">
-                    <a href="#story" class="nav__link">Our Story</a>
+                    <a href="/story" class="nav__link story" id="story-link">Our Story</a>
+                    <script>
+                        (function(){
+                            if (window.location.search === '?souls2thepolls') {
+                                var link = document.getElementById('story-link');
+                                link.setAttribute('href', '/story?souls2thepolls');
+                            }
+                        })();
+                    </script>
```

Comment:
Good catch making this link account for souls2thepolls.

Just a thought: Instead of including a bespoke script next to each link, what we could do here is add a script at the bottom to update every relative URL on the page. e.g. if using jQuery I'd do something like this:
```
if (window.location.search === '?souls2thepolls') {
    $('a[href^="/"]').each(function() {
        $(this).attr('href', $(this).attr('href') + '?souls2thepolls');
    });
}
```

What do you think?